packages/fxa-content-server/app/scripts/views/settings.js

```diff
@@ -109,6 +115,8 @@ const View = BaseView.extend({
     const account = this.getSignedInAccount();
 
     context.set({
+      ccExpired: !!this._ccExpired,
```

Comment:
If you manually set this to `true`, you'll see the problem I referred to above, `ccExpiredLinkAttrs` is an invalid variable name within `unsafeTranslate`

<img width="580" alt="Screenshot 2019-07-23 at 10 23 37" src="https://user-images.githubusercontent.com/848085/61700108-0bd62900-ad34-11e9-9b2e-672429421c27.png">